src/SystemProcessCounter.php

```diff
@@ -14,17 +14,34 @@ class SystemProcessCounter
     public static $command = 'horizon:work';
 
     /**
-     * Get the number of Horizon workers for a given supervisor.
+     * Get CPU and memory usage for each Horizon workers for a given supervisor.
      *
      * @param  string  $name
-     * @return int
+     * @return \Illuminate\Support\Collection
      */
-    public function get($name)
+    public function getWorkerStats($name)
     {
-        $process = Process::fromShellCommandline('exec ps aux | grep '.static::$command, null, ['COLUMNS' => '2000']);
+        $process = Process::fromShellCommandline('exec ps axo %cpu,%mem,cmd | grep '.static::$command.' | grep "supervisor='.$name.'" | grep -v "exec ps axo"', null, ['COLUMNS' => '2000']);
```

Comment:
`u` is removed as it formats the put put for human readability.
`%cpu,%mem,cmd` are all in the default output but being explicit both makes it easier to parse and reduces the amount of unused data being processed.

Reply:
This will not work on macOS:

```shell
$ exec ps axo %cpu,%mem,cmd

ps: cmd: keyword not found
```

On macOS, you should use `command` keyword.

Reply:
Thanks, I have changed it to `command` and tested that it still works on Linux.